addons/units/stringtable.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Units">
+        <Key ID="STR_ACE_Units_Module_DisplayName">
+            <English>Units</English>
+        </Key>
+
+        <Key ID="STR_ACE_Units_Speed_DisplayName">
+            <English>Speed</English>
+        </Key>
+        <Key ID="STR_ACE_Units_Speed_Description">
+            <English>Units used to display Speed</English>
+        </Key>
+
+        <Key ID="STR_ACE_Units_Alt_DisplayName">
+            <English>Altitude</English>
```

Comment:
```suggestion
            <English>Altitude</English>
            <German>Flughöhe</German>
```
There may be a more anglicized expression in the German Arma script, but I cba to check that out.

On the other hand, this may be a string in the game already, so I will probally check later.

Reply:
It wasn't one that I could find with https://synixebrett.github.io/stringtables. I used all the existing strings I found